backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/OutputFiles.kt

```diff
@@ -21,7 +21,12 @@ class OutputFiles(outputPath: String?, target: KonanTarget, produce: CompilerOut
     private val prefix = produce.prefix(target)
     private val suffix = produce.suffix(target)
 
-    val outputName = outputPath?.removeSuffixIfPresent(suffix) ?: produce.visibleName
+    val outputName: String
+
+    init {
+        val _outputName = outputPath?.removeSuffixIfPresent(suffix) ?: produce.visibleName
+        this.outputName = if (produce == CompilerOutputKind.FRAMEWORK) _outputName.capitalize() else _outputName
```

Comment:
What issue does this capitalization fix?

Reply:
#2358 🙏

Reply:
This appears to be the link to this pull request.
I'm trying to ask: why do you consider the capitalization required? What use-case does it help with?